tests/tnt/router_call_test.go

```diff
@@ -0,0 +1,52 @@
+package tnt_test
+
+import (
+	"context"
+	"log"
+	"math/rand"
+	"testing"
+	"time"
+
+	vshardrouter "github.com/KaymeKaydex/go-vshard-router"
+	"github.com/KaymeKaydex/go-vshard-router/providers/static"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/go-tarantool/v2/pool"
+)
+
+func TestRouterCall(t *testing.T) {
+	if !isCorrectRun() {
+		log.Printf("Incorrect run of tnt-test framework")
+		return
+	}
+
+	t.Parallel()
+
+	ctx := context.Background()
+
+	cfg := getCfg()
+
+	router, err := vshardrouter.NewRouter(ctx, vshardrouter.Config{
+		TopologyProvider: static.NewProvider(cfg),
+		DiscoveryTimeout: 5 * time.Second,
+		DiscoveryMode:    vshardrouter.DiscoveryModeOn,
+		TotalBucketCount: totalBucketCount,
+		User:             defaultTntUser,
+		Password:         defaultTntPassword,
+	})
+
+	if err != nil {
+		panic(err)
```

Comment:
lets not panic 
use require.NoError() instead

Reply:
Done